tasks/account.js

```diff
@@ -228,6 +228,62 @@ ACCOUNT_SCOPE.task("bot:remove", "Remove bot from the CMAccount")
         }
     });
 
+ACCOUNT_SCOPE.task("payment-token:add", "Add payment token to CMAccount")
+    .addOptionalParam("privateKey", "Private key to use", process.env.CMACCOUNT_PK)
```

Comment:
Why this is optional param? The privatekey is required to perform the transaction, isn't it?

Reply:
So it's possible to set the environment variable `CMACCOUNT_PK` and not provide the `--private-key` param from the command line. When you make it optional, you can provide a default value, and for that I've provided `process.env.CMACCOUNT_PK`.